Description: [PATCH] Remove django-discover-runner from test_require list in setup.py
Author: Ivan Udovichenko <iudovichenko@mirantis.com>
Forwarded: no
Date: Sat, 19 Sep 2015 01:33:54 +0300
Last-Update: 2015-09-22

diff --git a/setup.py b/setup.py
index c09af07..a7a358c 100644
--- a/setup.py
+++ b/setup.py
@@ -22,7 +22,6 @@ except ImportError:
 tests_require = [
     'Pillow',
     'django-compressor>=1.3',
-    'django-discover-runner',
     'mock',
 ]
 
-- 
2.1.4

